src/Components/Product.js

```diff
@@ -13,12 +14,36 @@ const Product = ({ product }) => {
   const removeFromCartHandler = () => {
     setIsInCart(false);
   };
+
+  const { wishlist, setWishlist } = useWishList();
+
+  const addWishlistHandler = (e) => {
+    e.preventDefault();
```

Comment:
Why `e.preventDefault()` on a button click?

Reply:
Thank you, made changes